src/google/protobuf/stubs/port.h

```diff
@@ -252,9 +252,13 @@ static const uint64 kuint64max = GOOGLE_ULONGLONG(0xFFFFFFFFFFFFFFFF);
 #define GOOGLE_GUARDED_BY(x)
 #define GOOGLE_ATTRIBUTE_COLD
 
+#ifndef GOOGLE_PROTOBUF_USE_UNALIGNED
```

Comment:
How about use GOOGLE_PROTOBUF_DONT_USE_UNALIGNED? It seems simpler if the user just need to define a macro than defining a macro to a specific value.